read-the-docs/2/conf.py

```diff
@@ -12,6 +12,12 @@ def parse_build_example_version():
       return os.environ['BUILD_EXAMPLE_VERSION']
     return 'master'
 
+def parse_branch_version():
+    import os
+    if "BRANCH_VERSION" in os.environ:
```

Comment:
you have introduce a new OS parameter. but I can not see it in PR of ivy-core

Reply:
That is done now, I changed the doc build Jenkinsfile and the release.groovy script.